### PR TITLE
Double Ice Cream, a new drink for bartenders to serve

### PR DESCRIPTION
**Pull request by dge21**
## About the PR

Added a new non alchoholic drink
## Why / Balance

its something I already used to mix, but people don't really pay attention to the "tastes like" messages, and this means the drink is more official looking and doesn't rely on me prompting them to pay attention to the taste like message
## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

probably not
**Changelog**

:cl:
- add: added a sweet new drink for the bar to serve

### CODE REVIEW

File: Resources/Prototypes/Flavors/flavors.yml [hunk 1]
```diff
@@ -1097,4 +1097,9 @@
 - type: flavor
   id: paintthinner
   flavorType: Complex
-  description: flavor-complex-paint-thinner
\ No newline at end of file
+  description: flavor-complex-paint-thinner
+
+  - type: flavor
```
Comment (deltanedas) [inline Resources/Prototypes/Flavors/flavors.yml:1102]: put this in deltav prototypes and indent properly

File: Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks.yml [hunk 2]
```diff
@@ -2344,3 +2344,19 @@
     - type: Icon
       sprite: Objects/Consumable/Drinks/bloodglass.rsi
       state: icon
+
+- type: entity
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks.yml:2348]: deltav prototypes

File: Resources/Locale/en-US/reagents/meta/consumable/drink/soda.ftl [hunk 3]
```diff
@@ -52,3 +52,5 @@ reagent-desc-fourteen-loko = A highly processed liquid substance barely-passing
 reagent-name-shamblers-juice = Shambler's Juice
 reagent-desc-shamblers-juice = ~Shake me up some of that Shambler's Juice!~
 
+reagent-name-double-ice-cream = Double Ice Cream
```
Comment (deltanedas) [inline Resources/Locale/en-US/reagents/meta/consumable/drink/soda.ftl:55]: deltav locale

File: Resources/Locale/en-US/flavors/flavor-profiles.ftl [hunk 4]
```diff
@@ -254,6 +254,7 @@ flavor-complex-themartinez = like violets and lemon vodka
 flavor-complex-cogchamp = like brass
 flavor-complex-white-gilgamesh = like lightly carbonated cream
 flavor-complex-antifreeze = warm
+flavor-complex-double-ice-cream = like ice cream, twice
```
Comment (deltanedas) [inline Resources/Locale/en-US/flavors/flavor-profiles.ftl:257]: deltav locale

File: Resources/Prototypes/Reagents/Consumable/Drink/soda.yml [hunk 5]
```diff
@@ -252,3 +252,19 @@
   physicalDesc: reagent-physical-desc-fizzy
   flavor: metallic
   color: "#66538F"
+
+- type: reagent
```
Comment (deltanedas) [inline Resources/Prototypes/Reagents/Consumable/Drink/soda.yml:256]: deltav prototypes

File: Resources/Prototypes/Recipes/Reactions/drinks.yml [hunk 6]
```diff
@@ -1117,3 +1117,15 @@
       amount: 1
   products:
     Cola: 4
+
+- type: reaction
```
Comment (deltanedas) [inline Resources/Prototypes/Recipes/Reactions/drinks.yml:1121]: deltav prototypes

File: Resources/Prototypes/Reagents/Consumable/Drink/soda.yml [hunk 7]
```diff
@@ -252,3 +252,19 @@
...
+  id: DoubleIceCream
+  name: reagent-name-double-ice-cream
+  parent: BaseSoda
+  desc: reagent-desc-double-ice-cream
+  physicalDesc: reagent-physical-desc-fizzy-and-creamy
+  flavor: doubleicecream
+  color: "#a06f42"
+  metamorphicSprite:
+    sprite: Objects/Consumable/Drinks/doubleicecreamglass.rsi
```
Comment (deltanedas) [inline Resources/Prototypes/Reagents/Consumable/Drink/soda.yml:265]: move it to deltav textures and update here

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by dge21** [comment]
got it, gonna do all that in a new pr because i don't know git well enough to edit one

**Comment by deltanedas** [comment]
you just push to the same branch you dont have to recreate it

**Pull request closed without merging**